### ISSUE TITLE
cloudflared has been updated to version ???

### ISSUE THREAD

**Issue by freddieleeman** [cloudflare/cloudflared#200]
As of version 2020.5.0 the `cloudflared update` command always returns 

> cloudflared has been updated to version

to stdout even when there is no update.

**Comment by FastHogi** [comment]
I can confirm this issue. I'm currently on 2020.5.1 and still the same behaviour. I have "cloudflared update > /dev/null" in a cron job and get an annoying e-mail notification with this message every day. Even with /dev/null doesn't resolved it, so it must be an output to stderr instead stdout.

**Comment by ymarkus** [comment]
Same here. Exit code of cloudflared update is `11`:
```bash
ymarkus@vps:~$ cloudflared update
cloudflared has been updated to version 
ymarkus@vps:~$ echo $?
11
```

**Comment by freddieleeman** [comment]
version 2020.6.1 still has same issue.

```
# cloudflared -version
cloudflared version 2020.6.1 (built 2020-06-09-2126 UTC)

# cloudflared update
cloudflared has been updated to version

# echo $?
11
```

**Comment by freddieleeman** [comment]
Looks like this issue is resolved in version 2020.6.5. Closing.

```
# cloudflared -v
cloudflared version 2020.6.5 (built 2020-06-17-1951 UTC)

# cloudflared update

# echo $?
0

```
